src/request.ts

```diff
@@ -272,6 +274,10 @@ export class OData {
     const { response: { headers } } = await this.fetchProxy(this.odataEnd, config);
     if (headers) {
       this.csrfToken = headers.get("x-csrf-token");
+
+      const cookiesObject =  ( headers as any )._headers['set-cookie']
+      this.sessionCookies = cookiesObject? Array.from( cookiesObject ).join(';') : this.sessionCookies
```

Comment:
access cookie by `document.cookie`

Reply:
fetch csrf token will not response `set-cookie` header if you have browser anything on the application server.

The cookie values will be response the first time user-agent access the site